acnportal/acnsim/base.py

```diff
@@ -652,7 +652,16 @@ def _from_registry(cls, in_registry, loaded_dict=None):
                         loaded_attr_value = attribute_dict[attr]
                     # If the attribute was originally JSON serializable,
                     # this is correct loading.
-                    setattr(out_obj, attr, loaded_attr_value)
+                    try:
+                        setattr(out_obj, attr, loaded_attr_value)
+                    except AttributeError:
```

Comment:
@zach401 This handles an error case I had missed before when a deserialized attribute corresponds to a property in a new implementation of the old object. This should be a pretty rare case, as it occurs only when making breaking changes of a certain type (e.g., in Unplug event, we have a property station_id where we once had an attribute).